app/components/SCForm.js

```diff
@@ -68,10 +65,17 @@ class SCForm extends Component {
     this.state = {
       value: {},
       renderPlaceholderOnly: true,
+      showErrors: false,
+      notANumErr: '',
+      overMaxErr: '',
+      overMinErr: '',
+      requireNumErr: '',
+      requireStrErr: '',
     };
 
     this.onChange = this.onChange.bind(this);
     this.onPress = this.onPress.bind(this);
+    this.handleBlur = this.handleBlur.bind(this);
```

Comment:
I was going to use it to check validation after the user clicks out of the box they are in.